airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/helpers/CdcConfigurationHelper.java

```diff
@@ -74,6 +76,33 @@ public static Optional<Integer> getFirstRecordWaitSeconds(final JsonNode config)
     return Optional.empty();
   }
 
+  private static Optional<String> getCdcServerTimezone(final JsonNode config) {
+    final JsonNode replicationMethod = config.get("replication_method");
+    if (replicationMethod != null && replicationMethod.has("server_time_zone")) {
+      final String serverTimeZone = config.get("replication_method").get("server_time_zone").asText();
+      return Optional.of(serverTimeZone);
+    }
+    return Optional.empty();
+  }
+
+  public static void checkCdcReplicationMethodConfig(final JsonNode config) {
+    checkFirstRecordWaitTime(config);
+    checkServerTimeZoneConfig(config);
+  }
+
+  @VisibleForTesting
+  public static void checkServerTimeZoneConfig(final JsonNode config) {
```

Comment:
If this method is `public`, would the need for @VisibleForTesting still be necessary? Unless my understanding is off, this usually for the usage of `default` or `protected` access modifiers, no? 

That said, I like the link out to the IANA website for customers to reference

Reply:
Removing the @VisibleForTesting